### ISSUE TITLE
tests broken regarding daylight saving time

### ISSUE THREAD

**Issue by roman-neuhauser** [python-babel/babel#156]
```
[   85s] + cd Babel-1.3
[   85s] + LC_ALL=C
[   85s] + py.test tests
[   85s] ============================= test session starts ==============================
[   85s] platform linux -- Python 3.4.1 -- py-1.4.26 -- pytest-2.7.0
[   85s] rootdir: /home/abuild/rpmbuild/BUILD/Babel-1.3, inifile: setup.cfg
[   86s] collected 323 items
[   86s] 
[   86s] tests/test_core.py ..............................................
[   87s] tests/test_dates.py .................F..F............................F..........
[   87s] tests/test_localedata.py ......
[   87s] tests/test_numbers.py ...........................
[   87s] tests/test_plural.py .......
[   87s] tests/test_support.py ..........................
[   87s] tests/test_util.py ..
[   87s] tests/messages/test_catalog.py ....................................
[   87s] tests/messages/test_checkers.py ......
[   87s] tests/messages/test_extract.py ....................................
[   87s] tests/messages/test_frontend.py ................................
[   87s] tests/messages/test_jslexer.py .
[   87s] tests/messages/test_mofile.py ...
[   87s] tests/messages/test_plurals.py .
[   87s] tests/messages/test_pofile.py ..................................
[   87s] 
[   87s] =================================== FAILURES ===================================
[   87s] ___________________ DateTimeFormatTestCase.test_timezone_gmt ___________________
[   87s] 
[   87s] self = <tests.test_dates.DateTimeFormatTestCase testMethod=test_timezone_gmt>
[   87s] 
[   87s]     def test_timezone_gmt(self):
[   87s]         tz = timezone('Europe/Berlin')
[   87s]         t = time(15, 30, tzinfo=tz)
[   87s]         fmt = dates.DateTimeFormat(t, locale='de_DE')
[   87s] >       self.assertEqual('GMT+01:00', fmt['ZZZZ'])
[   87s] E       AssertionError: 'GMT+01:00' != 'GMT+02:00'
[   87s] E       - GMT+01:00
[   87s] E       ?      ^
[   87s] E       + GMT+02:00
[   87s] E       ?      ^
[   87s] 
[   87s] tests/test_dates.py:188: AssertionError
[   87s] _________________ DateTimeFormatTestCase.test_timezone_rfc822 __________________
[   87s] 
[   87s] self = <tests.test_dates.DateTimeFormatTestCase testMethod=test_timezone_rfc822>
[   87s] 
[   87s]     def test_timezone_rfc822(self):
[   87s]         tz = timezone('Europe/Berlin')
[   87s]         t = time(15, 30, tzinfo=tz)
[   87s]         fmt = dates.DateTimeFormat(t, locale='de_DE')
[   87s] >       self.assertEqual('+0100', fmt['Z'])
[   87s] E       AssertionError: '+0100' != '+0200'
[   87s] E       - +0100
[   87s] E       ?   ^
[   87s] E       + +0200
[   87s] E       ?   ^
[   87s] 
[   87s] tests/test_dates.py:182: AssertionError
[   87s] ____________________________ test_get_timezone_gmt _____________________________
[   87s] 
[   87s]     def test_get_timezone_gmt():
[   87s]         dt = datetime(2007, 4, 1, 15, 30)
[   87s]         assert dates.get_timezone_gmt(dt, locale='en') == u'GMT+00:00'
[   87s]     
[   87s]         tz = timezone('America/Los_Angeles')
[   87s]         dt = datetime(2007, 4, 1, 15, 30, tzinfo=tz)
[   87s] >       assert dates.get_timezone_gmt(dt, locale='en') == u'GMT-08:00'
[   87s] E       assert 'GMT-07:00' == 'GMT-08:00'
[   87s] E         - GMT-07:00
[   87s] E         ?      ^
[   87s] E         + GMT-08:00
[   87s] E         ?      ^
[   87s] 
[   87s] tests/test_dates.py:384: AssertionError
[   87s] ===================== 3 failed, 320 passed in 2.05 seconds =====================
```

**Comment by warsaw** [comment]
I'm seeing these same failures when I build the Debian/Ubuntu version of the package.  Any thoughts on a fix?

**Comment by sils1297** [comment]
@roman-neuhauser @warsaw could you please give steps to reproduce? Which version/branch did you use?

**Comment by warsaw** [comment]
In Debian/Ubuntu we have version 1.3 currently, but it's a slightly modified version.  The package removes the bundled global.dat and localedata/*.dat and re-downloads and unpacks http://unicode.org/Public/cldr/23.1/core.zip.   Steps to reproduce reliably involve building the Debian package, but I believe the test failures are essentially this:

```
LC_ALL=C py.test
LC_ALL=C py.test-3
```

I've tried updating the Debian version to Babel 2.0, but I can't find a cldr version that works without throwing KeyErrors in scripts/import_cldr.py.  I tried everything newer than 23.1.

**Comment by etanol** [comment]
How did you execute the tests on version 2.0?  The test suite for the 2.0 branch was broken for some time, but it has been recently fixed (9f099d39553188d9bc56f7721ea40ece9f0959cd, aa4165df073239a9f6ff01c0b5f6cb430a5b9d10).  If you run tests with [Tox](http://tox.readthedocs.org/en/latest/), all of them should succeed.

**Comment by warsaw** [comment]
Thanks.  It was the aa1465d patch I was missing.

**Comment by etanol** [comment]
Closing, fixed in 2.1
